app/src/test/java/com/kickstarter/libs/utils/RewardViewUtilsTest.kt

```diff
@@ -42,6 +42,20 @@ class RewardViewUtilsTest : KSRobolectricTestCase() {
         assertEquals(R.string.Continue, RewardViewUtils.pledgeButtonText(backedProjectRwHasAvailableNotBackedAddOns, backedRwAvailableAddOns))
     }
 
+    /**
+     * Given a backedProject the backed reward
+     * has available addOns
+     * when not backed addOns
```

Comment:
what happens on android if you have backed add-ons in your original pledge?

Reply:
You can see the behaviour on the video from second 20 onwards

Reply:
ty! I see it!